frontend/lib/src/components/widgets/DataFrame/DataFrame.tsx

```diff
@@ -1242,8 +1242,8 @@ function DataFrame({
                 kind: "checkbox",
                 checkboxStyle: "square",
                 theme: {
-                  bgCell: gridTheme.glideTheme.bgHeader,
-                  bgCellMedium: gridTheme.glideTheme.bgHeader,
+                  bgCell: gridTheme.glideTheme.bgCell,
+                  bgCellMedium: gridTheme.glideTheme.bgCell,
```

Comment:
I'm not sure if we should do this here for dynamic editing. In this case, we are still only showing the checkboxes when they hover, so this will just be an empty column. Always showing the checkboxes puts quite a focus on the delete rows aspect.